clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/services/SubnetSelector.java

```diff
@@ -69,6 +70,20 @@ public SubnetSelector(
     return filteredSubnetIds;
   }
 
+  public Set<String> resolveSubnetsIdsForMultipleSubnetTypes(
```

Comment:
can we add a unit test for this function (on `SubnetSelector`)? Its return value is mocked in `CreateServerGroup...Spec`, so would like to ensure _this_ code is functioning as expected.

Reply:
Good catch! Will do!

Reply:
Addressed